__tests__/app.test.js

```diff
@@ -33,5 +33,65 @@ describe('app',() => {
             })
         })
     })
-    
+    describe('/api/reviews', ()=>{
+        test('200: responds with a joined array of review objects to comments table', ()=>{
+                return request(app)
+                .get('/api/reviews')
+                .expect(200)
+                .then(({body})=>{
+
+                    const reviewsV1 = body.reviews;
+
+                    // console.log(reviewsV1)
+
+                    expect(Array.isArray(reviewsV1));
+                    expect(reviewsV1).toHaveLength(13);
+                })
+
+        })
+        test('200: has correct field types and includes comment count field which joins from comments table. counting the amount of comments per review', ()=>{
+            return request(app)
+                .get('/api/reviews')
+                .expect(200)
+                .then(({body})=>{
+                const reviewsV1 = body.reviews;
+                reviewsV1.forEach((category)=>{
+                    expect(category).toHaveProperty('owner', expect.any(String));
+                    expect(category).toHaveProperty('title', expect.any(String));
+                    expect(category).toHaveProperty('review_id', expect.any(Number));
+                    expect(category).toHaveProperty('category', expect.any(String));
+                    expect(category).toHaveProperty('review_img_url', expect.any(String));
+                    expect(category).toHaveProperty('created_at', expect.any(String));
+                    expect(category).toHaveProperty('votes', expect.any(Number));
+                    expect(category).toHaveProperty('designer', expect.any(String));
+                    expect(category).toHaveProperty('comment_count', expect.any(String));
+                })
+            
+
+            })
+        })
+        test('200: sorts by date in descending order', ()=>{
+            return request(app)
+                .get('/api/reviews?sort_by=created_date')
+                .expect(200)
+                .then(({body})=>{
+                const reviewsV1 = body.reviews;
+
+                const sortedReviews = reviewsV1.sort((reviewsV1A, reviewsV1B)=>{
+                return reviewsV1A.created_at - reviewsV1B.created_at;
+               })
+                expect(sortedReviews).toEqual(reviewsV1);
```

Comment:
This is very brittle - I assume this is passing because the reviews have come back in the right order? As we're just comparing that they look the same as before they were sorted. Have a look at jest-sorted and see if you can find a way to implement this.